app/scripts/models/auth_brokers/fx-sync.js

```diff
@@ -25,31 +25,34 @@ define(function (require, exports, module) {
 
     type: 'fx-sync',
 
-    initialize (options = {}) {
-      this._metrics = options.metrics;
-      proto.initialize.call(this, options);
-    },
-
     afterSignUpConfirmationPoll (account) {
       return proto.afterSignUpConfirmationPoll.call(this, account)
         .then((defaultBehavior) => {
           if (this.hasCapability('cadAfterSignUpConfirmationPoll')) {
             // This is a hack to allow us to differentiate between users
             // who see CAD in the signup and verification tabs. CAD
-            // was added to the verifiation tab first, view names and view
+            // was added to the verification tab first, view names and view
             // events are all unprefixed. In the signup tab, we force add
             // the `signup` view name prefix so that events that contain
             // viewNames have `signup` view name prefix.
             //
             // e.g.:
             // screen.sms <- view the sms screen in the verification tab.
             // screen.signup.sms <- view the sms screen in the signup tab.
-            this._metrics.setViewNamePrefix('signup');
+            this.metrics.setViewNamePrefix('signup');
             return new ConnectAnotherDeviceBehavior(defaultBehavior);
           }
 
           return defaultBehavior;
         });
+    },
+
+    afterCompleteSignUp (account) {
+      return proto.afterCompleteSignUp.call(this, account)
+        .then((defaultBehavior) => {
+          this.metrics.setViewNamePrefix('');
```

Comment:
Where was this done before, and does it have a test? Document why `setViewNamePrefix` is being called.